hack/update-deps.sh

```diff
@@ -22,17 +22,17 @@ SERVING_VERSION="release-v1.2.0"
 # The list of dependencies that we track at HEAD and periodically
 # float forward in this repository.
 FLOATING_DEPS=(
-  "knative.dev/hack@v0.0.0-20220224013837-e1785985d364"
   "knative.dev/networking@${KN_VERSION}"
   "knative.dev/operator@${KN_VERSION}"
-  "knative.dev/pkg@v0.0.0-20220223180940-4fcbc1bc12e8"
 )
 
 FLOATING_FORK_DEPS=(
   "knative.dev/eventing-kafka-broker=github.com/openshift-knative/eventing-kafka-broker@${EVENTING_KAFKA_BROKER_VERSION}"
   "knative.dev/eventing=github.com/openshift/knative-eventing@${EVENTING_VERSION}"
   "knative.dev/eventing-kafka=github.com/openshift-knative/eventing-kafka@${EVENTING_KAFKA_VERSION}"
   "knative.dev/serving=github.com/openshift/knative-serving@${SERVING_VERSION}"
+  "knative.dev/pkg=knative.dev/pkg@${KN_VERSION}"
```

Comment:
This is supposed to be the openshift ones. What does it mean to move this here?

Reply:
Also the name now becomes a bit irrelevant.
Could be `FLOATING_DEPS_FORCED_VERSIONS`?

Reply:
This means force using whatever is in the `KN_VERSION` branch, so it fixes the fact that serving and eventing are at different hashes of these 2 packages.

I can introduce a different variable for these but the result is the same.

Reply:
Ok I guess the end result is the same hash though without requiring it upfront.

Reply:
or just `REPLACE_DEPS`

Reply:
but I think it's a very minor comment, I can do it next, it's not worth waiting another 4 hours for running e2e tests :)